assets/dev/scss/editor/panel/controls/_globals.scss

```diff
@@ -194,6 +194,7 @@
 					font-size: 13px;
 					content: '\e90e'; // eicon-check
 					text-shadow: 0px 0px 1px #000;
+					z-index: 1;
```

Comment:
?

Reply:
Since we added the transparency checkers background, the "selected color" "V" icon was not showing over the color preview box. This is because absolute positioning was used for the color box. This fixes it.